packages/calcite-components/src/components/action/action.tsx

```diff
@@ -211,6 +225,10 @@ export class Action extends LitElement implements InteractiveComponent, FormOwne
     }
   }
 
+  private storeIndicatorEl(el: HTMLDivElement): void {
```

Comment:
For consistency, can you use a `Ref`? Not critical for this PR. 

For future reference, our guidance will likely be to use refs unless additional logic is needed when the ref is updated.